chainer/distributions/log_normal.py

```diff
@@ -0,0 +1,91 @@
+import chainer
+from chainer.backends import cuda
+from chainer import distribution
+from chainer.functions.array import broadcast
+from chainer.functions.math import exponential
+import math
+
+LOGPROBC = - 0.5 * math.log(2 * math.pi)
+
+
+class LogNormal(distribution.Distribution):
+
+    """Logatithm Normal Distribution.
+
+    The probability density function of the distribution is expressed as
+
+    .. math::
+        p(x;\\mu,\\sigma) = \\frac{1}{\\sqrt{2\\pi\\sigma^2}x}
+            \\exp\\left(-\\frac{(\\log{x}-\\mu)^2}{2\\sigma^2}\\right)
+
+    Args:
+        loc(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
```

Comment:
In `scipy.stats`, the args `loc` and `scale` work on the final distribution.  The parameters of `LogNormal` could be just named `mu` and `sigma`.